app/templates/suppliers/_frameworks_live.html

```diff
@@ -29,8 +29,14 @@
         </p>
       {% endif %}
     {% endcall %}
+
     {% if framework.onFramework and not framework.needs_to_complete_declaration %}
-      {{ summary.edit_link('View documents and ask a question', url_for('.framework_dashboard', framework_slug=framework.slug)) }}
+        {% call summary.field() %}
+            {% if framework.framework == 'digital-outcomes-and-specialists' %}
+                <a href="{{ url_for('.opportunities_overview', framework_slug=framework.slug) }}">View your opportunities</a><br>
```

Comment:
This went through content. The empty text should be 'You haven't applied for any opportunities' So that will indicate what the page is for.